src/trace.ml

```diff
@@ -389,6 +389,16 @@ module Make_commands (Backend : Backend_intf.S) = struct
   let run_command =
     Command.async_or_error
       ~summary:"Runs a command and traces it."
+      ~readme:(fun () ->
+        "=== examples ===\n\n\
+         # Run a process, snapshotting at ^C or exit\n\
+         magic-trace run program -- arg1 arg2!\n\n\
```

Comment:
```suggestion
         magic-trace run ./program -- arg1 arg2\n\n\
```

And add ./ to other calls to [program] to make it clear that it's an exe.

Reply:
Good idea, done